vercel.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": 2,
+    "builds": [
+      {
+        "src": "app.py",
+        "use": "@vercel/python"
```

Comment:
Is your site deployed on Vercel? If yes please drop the link
I do not see a functional link on your repo

Reply:
Currently working on the deployment, will provide you the link soon .